tests/test_vectors.py

```diff
@@ -0,0 +1,170 @@
+import json
+import os
+
+from umbral.curvebn import CurveBN
+from umbral.point import Point, unsafe_hash_to_point
+from umbral.keys import UmbralPublicKey
+from umbral.config import default_params
+from umbral.fragments import KFrag, CapsuleFrag
+from umbral import pre
+
+def test_curvebn_operations():
+
+    vector_file = os.path.join('vectors', 'vectors_curvebn_operations.json')
+    try:
+        with open(vector_file) as f:
+            vector_suite = json.load(f)
+    except OSError:
+        raise 
```

Comment:
Is this a stub for future error handling?

Reply:
Yeah, not sure how we should handle the fact that the test vector fail doesn't exist. Perhaps simply fail the test?